tests/ListProjectsOptionsTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Sandboxes\Api\Tests;
+
+use Keboola\Sandboxes\Api\ListProjectsOptions;
+use PHPUnit\Framework\TestCase;
+
+class ListProjectsOptionsTest extends TestCase
+{
+    /**
+     * @dataProvider provideToQueryParametersTestData
+     */
+    public function testToQueryParameters(ListProjectsOptions $options, array $expectedResult): void
+    {
+        self::assertSame($expectedResult, $options->toQueryParameters());
+    }
+
+    public function provideToQueryParametersTestData(): iterable
+    {
+        yield 'empty' => [
+            ListProjectsOptions::create(),
+            [],
+        ];
+
+        yield 'hasPersistentStorage: true' => [
+            ListProjectsOptions::create()->setHasPersistentStorage(true),
+            ['hasPersistentStorage' => true],
+        ];
+
+        yield 'hasPersistentStorage: false' => [
+            ListProjectsOptions::create()->setHasPersistentStorage(false),
+            ['hasPersistentStorage' => false],
+        ];
```

Comment:
how about adding here one case for where `clearHasPersistentStorage` has been called?

Reply:
I don't think it belongs here. The `testToQueryParameters` tests if various states of the `ListProjectsOptions` are properly exported. And all possible state are already covered (`hasPersistentStorage: null/true/false`).

`set/get/clear` behaviour is tested by `testHasPersistentStorage`.